### ISSUE TITLE
Add attribute to questionnaire data about what "level" the question is at (e.g., main, roster)

### ISSUE THREAD

**Issue by arthur-shaw** [arthur-shaw/susopara#7]

**Comment by Shelmith-Kariuki** [comment]
Hi @arthur-shaw , did you get time to work on this?

**Comment by arthur-shaw** [comment]
No, I didn't. Let's come back to this when we've produced a minimum viable
product. I need to think about how this would be done, and what level of
effort would be required (read: how much I'd have to rewrite susopara).
